toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -272,23 +274,32 @@ func parseReleaseString(releaseInfo string) (packageRelease uint32, distroName s
 }
 
 func getPackageInformation(imageChroot *safechroot.Chroot, packageName string) (info packageInformation, err error) {
-	var packageVersion string
+	var packageInfo string
 	err = imageChroot.UnsafeRun(func() error {
-		packageVersion, _, err = shell.Execute("rpm", "-q", "--queryformat", "%{VERSION}", packageName)
+		packageInfo, _, err = shell.Execute("tdnf", "info", packageName, "--repo", "@system")
 		return err
 	})
 	if err != nil {
-		return info, fmt.Errorf("failed to query current package information for (%s):\n%w", packageName, err)
+		return info, fmt.Errorf("failed to query (%s) package information:\n%w", packageName, err)
 	}
 
-	releaseInfo := ""
-	err = imageChroot.UnsafeRun(func() error {
-		releaseInfo, _, err = shell.Execute("rpm", "-q", "--queryformat", "%{RELEASE}", packageName)
-		return err
-	})
-	if err != nil {
-		return info, fmt.Errorf("failed to query current package information for (%s):\n%w", packageName, err)
+	// Regular expressions to match Version and Release
+	versionRegex := regexp.MustCompile(`(?m)^Version\s+:\s+(\S+)`)
```

Comment:
nit: Put the regexes at the top of the file.

Reply:
cursious, why Chris though? Those are not constants that will be used in other places...